### PR TITLE
added one question relating binary and linear search

### PR DESCRIPTION
**Pull request by 836hardik-agrawal**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/general-cs-quiz.ts [hunk 1]
```diff
@@ -1282,6 +1282,18 @@ const generalCSQuiz = [
       "An O(branches ^ depth) runtime is often reported (although not always) for recursive functions that make multiple calls.",
     Link: "https://www.freecodecamp.org/news/learn-big-o-notation/"
   }
+   {
```
Comment (chefgs) [inline src/data/general-cs-quiz.ts:1285]: Linting failed. Please add comma at the end of previous line no 1284.

File: src/data/general-cs-quiz.ts [hunk 2]
```diff
@@ -1282,6 +1282,18 @@ const generalCSQuiz = [
...
+    Question:
+      "What is the worst case time complexity of binary search vs linear search?",
+    Answer: "Time complexity of binary search is O(logn) whereas time complexity of linear search is O(n)",
+    Distractor1: "Time complexity of binary search is O(n) whereas time complexity of linear search is O(logn)",
+    Distractor2: "Time complexity of binary search is O(n) whereas time complexity of linear search is O(n)",
+    Distractor3: "Time complexity of binary search is O(logn) whereas time complexity of linear search is O(logn)",
+    Explanation:
+      "In worst case in case of linear search we have iterate through n elements thus O(n)whereas in binary search in each step we divide in two so O(logn).",
+    Link: "https://www.freecodecamp.org/news/search-algorithms-linear-and-binary-search-explained/"
+  }
```
Comment (chefgs) [inline src/data/general-cs-quiz.ts:1295]: Linting failed. Please add comma at the end of line no 1295.

**Review by chefgs — COMMENTED** [review]
@836hardik-agrawal Please check the review comments, to resolve the liniting issues. Thanks!

**Comment by jdwilkin4** [comment]
I am closing this PR because it looks like you have open another one. 
But in the future, you don't need to open a new PR if you need to make changes.
You can push up your changes to the original PR instead 👍

**Pull request closed without merging**